twoliter/embedded/Makefile.toml

```diff
@@ -894,6 +894,29 @@ ln -snf "${BUILDSYS_VERSION_FULL}" "${OUTPUT_LOGS_DIR}/latest"
 '''
 ]
 
+[tasks.build-all]
+dependencies = ["fetch", "build-sbkeys", "publish-setup", "cargo-metadata"]
+script = [
+'''
+export PATH="${TWOLITER_TOOLS_DIR}:${PATH}"
+
+# Save built artifacts for each architecture.  We don't set this everywhere
+# because we build host tools with cargo as well, like buildsys and pubsys.
```

Comment:
This comment doesn't seem correct anymore, since we don't build host tools in Makefile.toml. Maybe we can just set `CARGO_TARGET_DIR` everywhere?

Reply:
> Maybe we can just set CARGO_TARGET_DIR everywhere?

Maybe, though there could be unexpected interactions with other uses of cargo, like `cargo test` in the unit tests task.